lib/logger.js

```diff
@@ -7,9 +7,10 @@ const { isFunction, isNil, isString } = require('lodash');
 class Logger {
   constructor(config) {
     const {
-      log: { debug, warn, error, deprecate, inspectionDepth } = {},
+      log: { debug, warn, error, deprecate, inspectionDepth, enableColors } = {},
     } = config;
     this._inspectionDepth = inspectionDepth || 5;
+    this._enableColors = isNil(enableColors) ? process.stdout.isTTY : enableColors;
```

Comment:
is `process.stdout.isTTY` available in browser? does this need defensive access chaining?